src/Docker.PowerShell/Cmdlets/ImportContainerImage.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.Threading.Tasks;
+using System.Management.Automation;
+using Docker.DotNet.Models;
+using System.IO;
+using Newtonsoft.Json;
+using System.Text;
+using System.Linq;
+using Docker.PowerShell.Objects;
+
+namespace Docker.PowerShell.Cmdlets
+{
+    [Cmdlet(VerbsData.Import, "ContainerImage",
+            DefaultParameterSetName = CommonParameterSetNames.Default)]
+    [Alias("Load-ContainerImage")]
+    [OutputType(typeof(ImagesListResponse))]
+    public class ImportContainerImage : DkrCmdlet
+    {
+        private const string _loadedImage = "Loaded image: ";
+
+        #region Parameters
+
+        [Parameter(Position = 0,
+            Mandatory = true)]
+        [ValidateNotNullOrEmpty]
+        public string[] FilePath { get; set; }
+
+        #endregion
+
+        #region Overrides
+
+        protected override async Task ProcessRecordAsync()
+        {
+            foreach (var item in FilePath)
+            {
+                var filePath = System.IO.Path.Combine(SessionState.Path.CurrentFileSystemLocation.Path, item);
+
+                string imageId = null;
+                bool failed = false;
+
+                // Ensure the path is a file.
+                if (!File.Exists(filePath))
+                {
+                    throw new FileNotFoundException(filePath);
+                }
+                var fi = new FileInfo(filePath);
+                using (var file = fi.OpenRead())
```

Comment:
`using (var file = File.OpenRead(filePath)`. Also pretty sure this throws FileNotFoundException if the path isnt valid so you likely dont need the check above either but you should check on that for me.